sme_finance_application_schema/sme_v5

```diff
@@ -299,13 +299,34 @@
                 "sage_live",
                 "sage_instant",
                 "sage_line_50",
-                "sage_other", 
-                "xero_quickbooks_online",
-                "xero_quickbooks_pro",
+                "sage_other",
+                "xero",
+                "quickbooks_online",
```

Comment:
We recognise this is a breaking change, however `sme_v5` was never a schema that was pushed for external use